views/cypress/tests/properties/simple.spec.js

```diff
@@ -265,9 +290,24 @@
             cy.assignValueToTextProperty(property, value);
         });
 
+        it('child item inherits parent property "Calendar" and sets value', function () {
+            const property = propertiesInfo.calendar;
+            const value = `2022-02-02 02:22`;
+
+            cy.assignValueToCalendarProperty(property, value);
+        });
+
+        it('child item inherits parent property "Password" and sets value', function () {
+            const property = propertiesInfo.password;
+            const value = `Cypress writing inside ${property.name}`;
+
+            cy.assignValueToTextProperty(property, value);
```

Comment:
Should not we extract it to a new command here to be consistent? 

https://github.com/oat-sa/tao-core/pull/3421/files#diff-940e964a3305823c71b2ff5fba42c6c571b37a148e244f1632294de7111ac10fR153